xml/System.Text.Json/Utf8JsonWriter.xml

```diff
@@ -719,14 +786,14 @@ The property name is escaped before writing.
           <format><![CDATA[
           
 The comment value is not escaped before writing.
-          
+
           ]]></format>
         </remarks>
         <exception cref="T:System.ArgumentException">The specified value is too large.
         
 -or-
 
-<paramref name="utf8Value" /> contains a comment delimiter (i.e. */).</exception>
+<paramref name="utf8Value" /> contains a comment delimiter (such as */).</exception>
```

Comment:
I don't think this should be "such as". There is only one valid comment delimiter that we look for.

```suggestion
<paramref name="utf8Value" /> contains a comment delimiter (that is, */).</exception>
```